vars/buildDockerAndPublishImage.txt

```diff
@@ -15,6 +15,7 @@ The following arguments are available for this function:
   * String **gitCredentials**: (Optional, defaults to "") If "automaticSemanticVersioning" is set, name of the credential to use when tagging the git repository. Support user/password or GitHub App credential types.
   * String **imageDir**: (Optional, defaults to ".", the parent directory of the Dockerfile) Path to a directory to use as build context (Example: "docker/", "python/2.7").
   * String **registry**: (Optional, defaults to "" which defines the registry based on the current controller setup) Container registry to deploy this image to (Example: "ghcr.io", "python/2.7").
+  * Boolean **unstash**: (Optional, default to "") Allow to restore files from a previously saved stash if not empty.
```

Comment:
If I understand correctly, this is not a boolean but a string representing the name of the previously saved stash : https://www.jenkins.io/doc/pipeline/steps/workflow-basic-steps/#unstash-restore-files-previously-stashed